common/static/common/js/spec/main_requirejs.js

```diff
@@ -1,6 +1,11 @@
-(function(requirejs, define) {
+/* globals _, requirejs */
+/* eslint-disable quote-props */
```

Comment:
Ah, that's interesting. I'll switch to the latest version and then see if I can remove these declarations. Thanks!